### PR TITLE
Merge hotfix branch.

### PR DESCRIPTION
**Pull request by charvolant**
Merge hotfix branch into develop. Partially so that import functionality can be tested without messing with the production name index. See #327

### CODE REVIEW

File: grails-app/services/au/org/ala/bie/IndexService.groovy [hunk 1]
```diff
@@ -30,18 +29,21 @@ class IndexService implements GrailsConfigurationAware {
...
 
     @Override
     void setConfiguration(Config config) {
         def search = config.solr.search
+        searchBoost = search.boost
+        searchDefType = search.defType
         searchTemplate = new SolrQuery()
         search.fq.each { searchTemplate.addFilterQuery(it) }
-        searchTemplate.set('defType', search.defType)
+        searchTemplate.set('defType', searchDefType)
```
Comment (Rita-C) [inline grails-app/services/au/org/ala/bie/IndexService.groovy:42]: Do you need to check null or the template works with null?

File: grails-app/services/au/org/ala/bie/IndexService.groovy [hunk 2]
```diff
@@ -30,18 +29,21 @@ class IndexService implements GrailsConfigurationAware {
...
         searchTemplate.set('qf', search.qf.join(' '))
         search.bq.each { searchTemplate.add('bq', it) }
         searchTemplate.set('q.alt', search.qAlt)
-        searchTemplate.set('boost', search.boost)
+        searchTemplate.set('boost', searchBoost)
```
Comment (Rita-C) [inline grails-app/services/au/org/ala/bie/IndexService.groovy:46]: Same question of checking null

File: grails-app/services/au/org/ala/bie/IndexService.groovy (hunk 1 above)
Comment (charvolant) [reply grails-app/services/au/org/ala/bie/IndexService.groovy:42]: These should never be null, the application.yml provides a default value if nothing is supplied in the configuration.

Comment (charvolant) [reply grails-app/services/au/org/ala/bie/IndexService.groovy:42]: See https://github.com/AtlasOfLivingAustralia/bie-index/blob/master/grails-app/conf/application.yml#L227

File: grails-app/services/au/org/ala/bie/IndexService.groovy (hunk 2 above)
Comment (charvolant) [reply grails-app/services/au/org/ala/bie/IndexService.groovy:46]: Same as above. See https://github.com/AtlasOfLivingAustralia/bie-index/blob/master/grails-app/conf/application.yml#L235

**Review by Rita-C — APPROVED** [review]

**Pull request merged**